File: app/src/main/java/pl/krystiankaniowski/weatherapp/data/openweathermap/model/Rain.java

package pl.krystiankaniowski.weatherapp.data.openweathermap.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Rain {

    @SerializedName("3h")
    @Expose
    private Integer _3h;

    public Integer get3h() {
        return _3h;
    }

}

File: app/src/main/java/pl/krystiankaniowski/weatherapp/data/openweathermap/model/Rain.kt
package pl.krystiankaniowski.weatherapp.data.openweathermap.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Rain {

    /** Rain volume for the last 3 hours */
    @SerializedName("3h")
    @Expose
    val _3h: Int? = null

}
